Duplicati/Server/WebServer/CaptchaUtil.cs

```diff
@@ -43,6 +43,16 @@ public static class CaptchaUtil
                 .Where(x => x != null)
                 .ToArray();
 
+        /// <summary>
+        /// Approximate the size in pixels of text drawn at the given fontsize
+        /// </summary>
+        private static int approxTextWidth(string text, FontFamily fontfamily, int fontsize) {

```

Comment:
We usually begin method names with a capital letter.  Would you mind renaming this for consistency?

Also, the opening `{` typically appears on the next line.